tests/ga/test_extension.py

```diff
@@ -2168,8 +2168,10 @@ def test_ext_sequence_no_should_be_set_from_within_extension(self, *args):
                 printenv | grep ConfigSequenceNumber
                 """
 
-        base_dir = os.path.join(conf.get_lib_dir(), 'OSTCExtensions.ExampleHandlerLinux-1.0.0', test_file_name)
-        self.create_script(test_file_name, test_file, base_dir)
```

Comment:
AgentTestCase had drifted from its original implementation and some of the parameters were confusing. I did some simplification of the method's  signature.